injector/cpuset_manager.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Why is this file not part of the `stress` package? It sounds like it is related to the stresser itself.

Reply:
it can't be part of `stress` package due to cyclic dependency.

you'll see that this `StressManager` needs to operate on `injector` (parse cores and update GOMAXPROCS). `injector` needs to share a `StressManager` instance. Hence the cycle

Reply:
What if instead of providing `config CPUPressureInjectorConfig` to `TrackInjectorCores` you provide `config.CGroup` to it? It seems it's enough to properly resolves this issue and having a clean package responsible.

It would remove the cyclic dependency.

And in fact, `StressManager` feels to need to leave into our `stress` package.

Reply:
good point, i think it's correct, let me rework the parameter.